cpp/data/system_updates/146.ixtheo.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE keyword_translations MODIFY COLUMN status enum ('reliable_synonym','reliable','unreliable', 'unreliable_cat2', 'unreliable_synonym','replaced','replaced_synonym','new','new_synonym');
```

Comment:
Is 'macs' missing here as possible enum value? (it is still present if you compare it with installer/ixtheo.sql)